src/node.rs

```diff
@@ -238,25 +237,24 @@ pub struct Element {
 
 impl Element {
     #[doc(hidden)]
-    pub fn new(name: QualName, attrs: Vec<Attribute>) -> Self {
-        let id = attrs
-            .iter()
-            .find(|a| a.name.local.deref() == "id")
-            .map(|a| LocalName::from(a.value.deref()));
-
-        let classes: HashSet<LocalName> = attrs
-            .iter()
-            .find(|a| a.name.local.deref() == "class")
-            .map_or_else(HashSet::new, |a| {
-                a.value
-                    .deref()
-                    .split_whitespace()
-                    .map(LocalName::from)
-                    .collect()
-            });
+    pub fn new(name: QualName, attributes: Vec<Attribute>) -> Self {
+        let mut classes: HashSet<LocalName> = HashSet::new();
+        let mut id: Option<LocalName> = Option::default();
```

Comment:
I think `None` would be less surprising when reading this?

Reply:
updated